src/modules/peek/Peek.UI/MainWindow.xaml.cs

```diff
@@ -89,6 +89,7 @@ private void Uninitialize()
 
             ViewModel.Uninitialize();
             ViewModel.ScalingFactor = 1;
+            FilePreviewer.ResetSource();
```

Comment:
This won't close the video when navigating away from it.

Reply:
> partial void OnPreviewerChanging(IPreviewer? value)
Already pause the video perhaps we need to reset there instead or reset and pause perhaps.

Reply:
Done. Can you re-check? Thanks :)